src/std/env.ab

```diff
@@ -74,3 +75,53 @@ pub fun is_root(): Bool {
 
     return false
 }
+
+pub fun printf(format: Text, args: [Text] = [""]): Null {
+    unsafe ${nameof args}=("{format}" "\$\{{nameof args}[@]}")$
+    unsafe $printf "\$\{{nameof args}[@]}"$
+}
+
+pub fun printf_escape(text: Text): Text {
+    return unsafe $echo \${nameof text}|sed -e 's/\\\\/\\\\\\\\/g' -e "s/%/%%/g"$
+}
+
+pub fun text_shell(message: Text, color1: Num, style: Num, color2: Num): Text {
+    return "\e[{style as Text};{color1 as Text};{color2 as Text}m{printf_escape(message)}\e[0m"
+}
+
+pub fun text_bold(message: Text): Text {
+    return "\e[1m{printf_escape(message)}\e[0m"
+}
+
+pub fun text_italic(message: Text): Text {
+    return "\e[3m{printf_escape(message)}\e[0m"
+}
+
+pub fun text_underlined(message: Text): Text {
+    return "\e[4m{printf_escape(message)}\e[0m"
+}
+
+pub fun color_echo(message: Text, color: Num): Null {
+    printf("\e[{color as Text}m%s\e[0m\n", [message])
+}
+
+pub fun echo_info(message: Text): Null {
+    printf("\e[1;3;97;44m %s \e[0m\n", [message])
+}
+
+pub fun echo_success(message: Text): Null {
+    printf("\e[1;3;97;42m %s \e[0m\n", [message])
+}
+
+pub fun echo_warning(message: Text): Null {
+    printf("\e[1;3;97;43m %s \e[0m\n", [message])
+}
+
+pub fun error(message: Text, exit_code: Num = 1): Null {
+    printf("\e[1;3;97;41m %s \e[0m\n", [message])
+    if exit_code > 0 : exit(exit_code)
+}
```

Comment:
Could you change the `\e` to `\033`? Just so that everything is uniform

Reply:
`\033` not work, i think is a rust problem.
`Error: Error { kind: InvalidInput, message: "nul byte found in provided data" }`
I used the hexadecimal notation instead